src/RecoveryServices/RecoveryServices.Backup.Autorest/custom/Helpers/PolicyHelpers.ps1

```diff
@@ -802,4 +802,80 @@ function ValidateMandatoryFields {
             }
         }
     }
+}
+
+
+
+function ValidateTieringPolicy
+{
+    if($tieringdetails.TieringMode -eq "")
+    {
+        $errormsg="Please specify the tiering mode"
+        throw $errormsg
+    }
+    if(($tieringdetails.TieringMode -eq "DoNotTier") -or ($tieringdetails.TieringMode -eq "TierRecommended"))
+    {
+        if(($tieringdetails.Duration -ne $null -or $tieringdetails.durationType -ne $null) -and ($tieringdetails.Duration -gt 0 -and $tieringdetails.durationType -ne "Invalid"))
+        {
+            $errormsg="Invalid values provided according to TieringMode"
```

Comment:
can we improve this error message to be more accurate. something like 

Invalid parameters TierAfterDuration, TierAfterDurationType for the given TieringMode